two_factor/utils.py

```diff
@@ -61,3 +62,14 @@ def totp_digits():
     for totp tokens. Defaults to 6
     """
     return getattr(settings, 'TWO_FACTOR_TOTP_DIGITS', 6)
+
+
+def user_is_anonymous(user):
+    if not user:
+        return True
+
+    # https://docs.djangoproject.com/en/dev/releases/1.10/#using-user-is-authenticated-and-user-is-anonymous-as-methods
+    if django.VERSION < (2, 0):
```

Comment:
Could this be changed to `< (1, 10)`? That will prevent DeprecationWarnings being emitted on Django 1.10 and 1.11.